### DISCUSSION TITLE
database of serenity didnt work

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5601] [Help]
hi everyone 
now i create new project with serenity and didn't add anything yet just want it run with its default component 
i get error the they cannot find the database 
how can i solve this error 

![Screenshot (1)](https://user-images.githubusercontent.com/77009503/112748907-370c2380-8fbf-11eb-8a64-be63369fbc56.png)

**Comment by VictorTomaili** [comment]
It's impossible to know your database connection string for us. Please ensure you can connect your database with other applications like SSMS etc. Then ensure your connection string is valid.

**Reply by samaremad** [reply]
no dear i didnt add any database 
this error for database of serenity in local server
 it is dosnt work  

and this is the connection string in web config

`  <connectionStrings>
    <add name="Default" connectionString="Data Source=(LocalDb)\MSSqlLocalDB; Initial Catalog=Serene5_Default_v1; Integrated Security=True" providerName="System.Data.SqlClient" />
  </connectionStrings>
`

**Reply by VictorTomaili** [reply]
Because you didn't have a local database installed in your computer. Search MsSql LocalDb on google and install.

**Reply by samaremad** [reply]
i make sure that local database is installed and founded in the system but the same problem still appear i think it is need some configuration in the visual studio 
what should i do

**Reply by VictorTomaili** [reply]
search localdb and how to use it on google
